### PR TITLE
Feeder Structure

### PR DESCRIPTION
**Pull request by AntarcticaByToto**

### CODE REVIEW

**Comment by AntarcticaByToto** [comment]
Set up code structure for feeder subsystem.

File: Competition/src/main/cpp/subsystems/Feeder.cpp [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+        }
+    }
+    else {
+        state.feederState = FeederState::FEEDER_DISABLE;
+    }
+}
+
+void Feeder::analyzeDashboard()
+{
+    // @TODO Reverse speed set via dashboard
```
Comment (mray190) [inline Competition/src/main/cpp/subsystems/Feeder.cpp:85]: Use smart dashboard to set some speeds

File: Competition/src/main/cpp/subsystems/Feeder.cpp [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    // @TODO Intake1/Intake2 speed set via dashboard
+}
+
+void Feeder::assignOutputs()
+{
+    // @TODO set variables for the speeds instead of calling set every if statement
```
Comment (mray190) [inline Competition/src/main/cpp/subsystems/Feeder.cpp:91]: Set variables for the speeds, call set at the end of the function

**Review by mray190 — CHANGES_REQUESTED** [review]
See comments below

**Pull request closed without merging**